include/pros/apix.h

```diff
@@ -550,7 +666,110 @@ int32_t fdctl(int file, const uint32_t action, void* const extra_arg);
  */
 #define DEVCTL_SET_BAUDRATE 17
 
+
+/******************************************************************************/
+/**                       RTOS STATIC TASK FACILITIES                        **/
+/******************************************************************************/
+
+#ifdef __cplusplus
+#define task_t pros::task_t
+#define task_fn_t pros::task_fn_t
+#define mutex_t pros::mutex_t
+#define sem_t pros::c::sem_t
+#define queue_t pros::c::queue_t
+#endif
+
+typedef uint32_t task_stack_t;
+
+/**
+ * Creates a task using statically allocated buffers. All tasks used by the PROS
+ * system must use statically allocated buffers.
```

Comment:
Instead of saying PROS systems must use buffers, I'd recommend saying the advantages of static tasks instead.

Reply:
This applies to all of these types.

Reply:
I see. I wonder if you could help me rewrite the comment since you are familiar with this topic.

Reply:
Maybe something like: 
Creates a task using pre-allocated buffers. Given valid inputs, the task will be created.
That's kind of what the FreeRTOS headers say.